meta-genivi-dev/recipes-extended/chromium/chromium-wayland_%.bbappend

```diff
@@ -5,6 +5,9 @@ SRC_URI += "\
     file://chromium.service \
 "
 
+# FIXME: workaround for https://at.projects.genivi.org/jira/browse/LM-2
+CHROMIUM_EXTRA_ARGS_append = " --window-size=1728,1080"
+
```

Comment:
Good, we have a workaroud but I understand you couldn't build for Silk yet so i guess it's not be tested on more than 1 target and monitor?